app/assets/javascripts/mumuki_laboratory/application/codemirror.js

```diff
@@ -71,8 +71,6 @@ var mumuki = mumuki || {};
 
 
   mumuki.load(function () {
-    CodeMirror.currentLocale = mumuki.locale;
```

Comment:
Why are you removing this line?

Reply:
This affects not only gobstones but all the highlighters

Reply:
Gobstones was the only mode using it. It was added in https://github.com/mumuki/mumuki-laboratory/pull/1226 but it doesn't work well with the new mode so I've removed it.

Reply:
OK!